dev_tools/requirements/deps/nbformat.txt

```diff
@@ -0,0 +1,2 @@
+# TODO: add latest commit retrieval to compile.py - pip-compile can't upgrade automatically
```

Comment:
what does this mean?

Reply:
oh, I'll remove this. There was an iteration of this code which used pip-compile to generate the .txt-s from .in files - but it grew way too expensive so I got rid of it.